### ISSUE TITLE
Update needed for Technique ARIA22 example "Using role=status to Provide Status Message on Shopping Cart"?

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag#1137]
Hi,
It seems that the example referenced in Technique [ARIA22 Using role=status to Provide Status Message on Shopping Cart](https://www.w3.org/WAI/WCAG21/working-examples/aria-role-status-shoppingcart/) does not work, maybe because of changes to the claimed default value `aria-atomic` of `role=status`.
According to the Technique, the output should be "Shopping Cart # items" or "# items".

```
<p role="status"><img src="shopping-cart.png" alt="Shopping Cart">
<span id="cart">1</span> items</p>
```
Current output however is:
---> NVDA/FF: exposes number only
---> JAWS/FF: no output
---> JAWS/Chrome: output "2 items"
---> VO/iOS: output "shopping cart 2 items"

Adding `aria-atomic="true"` after `role="status"`  in the devtools achieves the desired output "Shopping cart 2 items" (only tested in NVDA/FF).

**Comment by JAWS-test** [comment]
The support of the alt attribute within Live Region is not good: https://github.com/FreedomScientific/VFO-standards-support/issues/299

**Comment by alastc** [comment]
Is this a straightforward update to the technique? If someone can make a PR, it looks like a simple thing to approve.
